app/routes/index.js

```diff
@@ -7,10 +7,14 @@ const fxPromo = require('../templates/fxPromo');
 
 const app = choo();
 
+function showBanner(route) {
+  return !route.startsWith('/unsupported/');
```

Comment:
we still want to check `state.promo` so `state.promo &&`

Reply:
> we still want to check state.promo.

Sorry, I missed upon that in a hurry

Reply:
I have made the changes